raw/export_to_sparql.ipynb

```diff
@@ -1,561 +0,0 @@
-{
```

Comment:
why was this file deleted?

Reply:
I thought it, it does not necessarily belong to the cleanup pipeline. It is a totally different concern, i.e. constructing the data-model and generating meta-data for SPARQL import, as far as I understand. If we decide to make the cleanup pipeline and the SPARQL export pipeline into a single entity I will work on finding a place in the workflow.